TypeScript Error: Interface 'SSEResponse' incorrectly extends interface 'Response'.

## Bug Report

**Current Behavior**

Our builds started to fail, and I'm not sure why:
https://travis-ci.org/probot/probot/jobs/606128221#L244-L255

**Environment**
- Probot version(s): master
- Node/npm version: 8, 10
- OS: Travis CI (Ubuntu)

**Possible Solution**

I think `@types/node` was recently updated, maybe it's related to that?

**Additional context/Screenshots**

```
$ npm test

> probot@0.0.0-development test /home/travis/build/probot/probot

> tsc --noEmit -p test && jest --coverage && npm run lint && npm run doc-lint

test/webhook-proxy.test.ts:13:11 - error TS2430: Interface 'SSEResponse' incorrectly extends interface 'Response'.

  Types of property 'json' are incompatible.

    Type '(body: any, status?: string | undefined) => Response' is not assignable to type 'Send<this>'.

      Type 'Response' is not assignable to type 'this'.

13 interface SSEResponse extends Response {

             ~~~~~~~~~~~
```

--- a/package.json
+++ b/package.json
@@ -93,7 +93,7 @@
     "@types/cache-manager": "^2.10.0",
     "@types/dotenv": "^8.2.0",
     "@types/eventsource": "^1.1.0",
-    "@types/express": "^4.0.39",
+    "@types/express": "^4.17.2",
     "@types/ioredis": "^4.0.6",
     "@types/jest": "^24.0.9",
     "@types/js-yaml": "^3.10.1",

--- a/test/webhook-proxy.test.ts
+++ b/test/webhook-proxy.test.ts
@@ -11,7 +11,7 @@ import { createWebhookProxy } from '../src/webhook-proxy'
 const targetPort = 999999
 
 interface SSEResponse extends Response {
-  json (body: any, status?: string): Response
+  json (body: any, status?: string): this
 }
 
 describe('webhook-proxy', () => {

